### PR TITLE
Adding check only active programs can generate credentials.

### PR DESCRIPTION
**Pull request by awais786**
Adding checks so that only active ProgramCertificate configurations can generate credentials. I have added check for CourseCertificates too.
https://openedx.atlassian.net/browse/ECOM-3858

### CODE REVIEW

**Comment by ahsan-ul-haq** [comment]
Please write test to verify the behavior for inactive course credentials as well.

**Comment by awais786** [comment]
@ahsan-ul-haq courses credentials not implemented yet. So we don't need to write any tests cases for that.

**Comment by ahsan-ul-haq** [comment]
:+1:

**Comment by awais786** [comment]
@clintonb Please review this PR.

File: credentials/apps/api/tests/test_views.py [hunk 1]
```diff
@@ -406,6 +406,33 @@ def _assert_usercredential_fields(self, response, username, expected_attrs):
...
+            },
+            "attributes": [
+                {
+                    "name": self.user_credential_attribute.name,
+                    "value": self.user_credential_attribute.value
+                },
+            ]
+        }
+
+        msg = "Credential ID [{cred_id}] for [ProgramCertificate matching query does not exist.]".format(
```
Comment (clintonb) [inline credentials/apps/api/tests/test_views.py]: No need for the second set of brackets.

**Comment by clintonb** [comment]
:+1:

**Pull request merged**